auction-app-frontend/src/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+
+import "./style.scss";
+
+const Checkbox = ({ label, onChange }) => {
+  const [checked, setChecked] = useState(false);
+
+  const handleCheckboxChange = () => {
+    setChecked(!checked);
+    onChange(!checked);
```

Comment:
If I'm following here, you have state in parent component and you are changing it with onChange method. 
But you also have checked state here. 
Why not pass prop for checked from parent component.